### PR TITLE
install: add snap installation guide

### LINKED ISSUE #274: Add snappy installation instructions
Since we provide Kata as a snap...

- https://snapcraft.io/kata-containers

... we need some instructions explaining how to install on a system running the snapd package manager.

### PR DESCRIPTION
**Pull request by devimc**
Add snap installation guide

fixes #274

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: install/snap-installation-guide.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Install Kata Containers from snapcraft.io
+
+Kata Containers can be installed in any Linux distribution that supports [snapd][1].
+
+Run following command to install Kata Containers:
```
Comment (jodh-intel) [inline install/snap-installation-guide.md]:
```suggestion
Run the following command to install Kata Containers:
```

**Comment by marcov** [comment]
Thanks @devimc. This new document should be also added as a new installation method [in this table](https://github.com/kata-containers/documentation/blob/master/install/README.md#installing-on-a-linux-system).

Something like:
- installation method: using snapd
- suggested  for: ?? :)
- description: install kata easily on distributions where kata is not yet packaged

File: install/snap-installation-guide.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Install Kata Containers from snapcraft.io
+
+Kata Containers can be installed in any Linux distribution that supports [snapd][1].
```
[the remark this answers is not in the document]
Comment (devimc) [reply install/snap-installation-guide.md]: > Or maybe this is too obvious?

hehe, I think so :smile:

**Comment by jodh-intel** [comment]
Nice! I think we can fast-track this doc PR too fwiw...

lgtm

**Comment by marcov** [comment]
lgtm

**Comment by marcov** [comment]
We need one more review here, pinging @kata-containers/documentation

**Comment by jodh-intel** [comment]
This is a simple doc and the team have had ample time to comment. Merging...

**Pull request merged**